test/format/format.sh

```diff
@@ -467,10 +467,13 @@ resolve()
 		# give the parent recording binary a chance to complete if we are using it
 		[[ ! -z $live_record_binary ]] && sleep 2
 
-		# Check for Sanitizer failures, have to do this prior to success because both can be reported.
-		grep -E -i 'Sanitizer' $log > /dev/null && {
-			report_failure $dir
-			continue
+		# Process group leader dumped core: so it is a bug, in contrast to spurious cores
```

Comment:
Out of curiosity as I like the idea of checking for cores:
If `abort_on_error=1` is set, the process is killed after MSan has detected an error. Wouldn't it be caught with the code further down line ~ 530 which processes the different signals received?

Reply:
It should, but we had seen cases where it didn't report correctly but a core was produced. This is for cover as I understand it.

Reply:
Yes. Let me know if I can make this clearer in the comment. I wasn't able to reproduce the behavior but had limited time.